.github/workflows/codeql-analysis.yml

```diff
@@ -1,48 +0,0 @@
-# For most projects, this workflow file will not need changing; you simply need
-# to commit it to your repository.
-#
-# You may wish to alter this file to override the set of languages analyzed,
-# or to provide custom queries or build logic.
-#
-# ******** NOTE ********
-# We have attempted to detect the languages in your repository. Please check
-# the `language` matrix defined below to confirm you have the correct set of
-# supported CodeQL languages.
-#
-name: "CodeQL"
-
-on:
-  push:
-    # The branches below must be a subset of the branches above
-    branches: "**"
-  pull_request:
-    # The branches below must be a subset of the branches above
-    branches: [ main ]
-  schedule:
-    - cron: '0 0 * * 0'
-  workflow_dispatch:
```

Comment:
>workflow_dispatch: [](http://example.com/codeflow?start=2&length=18)

suggestion: instead of remove, how about change it to on demand only? That way we can test if need.

Reply:
the reason is that I have no motivation to spend more time on this topic than I have. if you have a commit to propose you think will do what you suggest, contribute it here. otherwise this is going away because i have lots of other stuff competing for attention. :) i suspect you can empathise.

Reply:
yes! I know `workflow_dispatch` is the on demand one so I quickly changed it. I think ready to merge now.